cmd/apps/inletsoperator_app.go

```diff
@@ -235,9 +257,17 @@ func getInletsOperatorOverrides(command *cobra.Command) (map[string]string, erro
 		}
 		overrides["organization-id"] = orgID
 
+		if len(secretKeyFile) == 0 {
```

Comment:
This looks like it would fail all the time? Should it be inside an if statement?

Reply:
if its empty, dont add the override, if its not empty, (i have set --secret-key-file) then add the override